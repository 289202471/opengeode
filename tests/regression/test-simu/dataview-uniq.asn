TASTE-BasicTypes DEFINITIONS ::=
BEGIN
T-UInt32 ::= INTEGER (0..4294967295)
TASTE-Peek-id ::= INTEGER (0..4294967295)

TASTE-Peek-id-list ::= SEQUENCE (SIZE (1..10)) OF TASTE-Peek-id
FixedIntList ::= SEQUENCE (SIZE(3)) OF TASTE-Peek-id

MyEnum ::= ENUMERATED {one, two, three, four, five}

END
 


