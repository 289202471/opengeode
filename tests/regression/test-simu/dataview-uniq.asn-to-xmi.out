[INFO] Starting to parse model ./test-simu/dataview-uniq.asn
[ERROR] visitType: Unable to process: ENUMERATED{one,two,three,four,five}
[ERROR] (visitTypeAssigment) Unsupported construct:  Input: 'MyEnum ::= ENUMERATED {one, two, three, four, five}' at pos 9:0
  	 File: ./test-simu/dataview-uniq.asn
[ERROR] visitType: Unable to process: ENUMERATED{aa,bb}
[ERROR] visitType: Unable to process: CHOICE{aBOOLEAN,bENUMERATED{aa,bb}}
[ERROR] (visitTypeAssigment) Unsupported construct:  Input: 'MyChoice ::= CHOICE { a BOOLEAN, b ENUMERATED { aa, bb } }' at pos 11:0
  	 File: ./test-simu/dataview-uniq.asn
[ERROR] visitType: Unable to process: SET(SIZE(1..3))OFINTEGER(0..10)
[ERROR] (visitTypeAssigment) Unsupported construct:  Input: 'MySetOf ::= SET (SIZE(1..3)) OF INTEGER (0..10)' at pos 15:0
  	 File: ./test-simu/dataview-uniq.asn
[INFO] Parsing completed
java.lang.RuntimeException: Errors have occured during the tool's execution. The tool will be stopped.
	at ee.ibk.sdl.parser.utils.EventHandler.stopOnErrors(EventHandler.java:258)
	at ee.ibk.sdl.parser.Main.processInputFiles(Main.java:491)
	at ee.ibk.sdl.parser.Main.doMain(Main.java:190)
	at ee.ibk.sdl.parser.Main.main(Main.java:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:58)
